playbooks/webapp/tasks/create_aurora_setup.yaml

```diff
@@ -0,0 +1,181 @@
+---
+- name: Set 'region' variable
+  ansible.builtin.set_fact:
+    region: "{{ region | default(aws_region) }}"
+
+- name: Create resources playbook
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key | default(omit) }}"
+      aws_secret_key: "{{ aws_secret_key | default(omit) }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ region }}"
+  block:
+    - name: Get image ID to create an instance
+      amazon.aws.ec2_ami_info:
+        filters:
+          architecture: x86_64
+          virtualization-type: hvm
+          root-device-type: ebs
+          name: "{{ image_filter }}"
```

Comment:
all theses appear as variables which should be reference in the playbook doc. 
You should rather create a role and reference its README into the playbook doc

Reply:
With the exception of "image_filter," all other values remain constant. Therefore, I preferred not to retrieve them from the vars file. By "create a role and reference its README in the playbook doc" - which specific functionality or feature necessitates the creation of a role in this playbook? Can you please make it clear to me?

Reply:
I have changed it to a single create playbook now.